### ISSUE TITLE
No links to example videos

### ISSUE THREAD

**Issue by fish2000** [LLNL/TourRobotDance#2]
The `TourRobotDance` project’s README and GitHub metadata lack any links to videos (or GIFs, or other such trans-ephemeral documentation) of tape library robots dancing – thus depriving the overwhelming percentage of the GitHub community who lack the professional connections that could possibly grant an audience for a display as notionally thrilling as this, of any sort of concrete resolution for the piqued appetite for robot-dance displays that this projects’ description invariably inflames in the nerdly-inclined reader.

Please, don’t leave us stranded out here in the robotic equivalent of the town from “Footloose” – add a clarifying link to an example robot dance performance.

Thanks!

**Comment by IanLee1521** [comment]
Let's see what we can do!

@cleary3, do we have something like this?

**Comment by cleary3** [comment]
No, we have no such media materials. Our tape libraries are located in areas that make filming and photography difficult. Nonetheless, for those organizations that have SL8500 tape libraries and give tours that include the libraries on the tour route, it will be straightforward for them to imagine what a robot dance might entail and how it might enliven their data center tours (even without multimedia in the project README).
